op-acceptance-tests/tests/base/chain/unsafe_sync_test.go

```diff
@@ -0,0 +1,86 @@
+package chain
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
+	"github.com/stretchr/testify/require"
+)
+
+// TestUnsafeSync tests that the verifier nodes are able to sync the unsafe head of the chain.
+// It does this by waiting for the sequencer to produce NUM_UNSAFE_BLOCKS unsafe blocks and requiring the verifier nodes to
+// be progress to the final unsafe head number.
+func TestUnsafeSync(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewSingleChainMultiNode(t)
+
+	// Stop batcher so there is no sync from deriving from L1
+	// This won't work on external networks.
+	sys.L2Batcher.Stop()
+
+	l2chainID := sys.L2Chain.Escape().ChainID()
+	const NUM_UNSAFE_BLOCKS = 10
+	const TIMEOUT = 5 * time.Second // time to wait for all verifier nodes to catch up to the sequencer
+
+	// In order for this test to be valid, we need at least 2 L2 EL nodes.
+	t.Require().Greater(len(sys.L2Chain.L2ELNodes()), 1, "expected at least 2 L2 EL nodes")
```

Comment:
This should already be guaranteed by the preset when `DEVNET_EXPECT_PRECONDITIONS_MET` is `true` (the default).